### ISSUE TITLE
audit-avt1-guidelines-pictograms: img needs alt attribute

### ISSUE THREAD

**Issue by andreancardona** [carbon-design-system/carbon-website#1502]
## Environment:
> Operating system: Mac
> Browser: Chrome
> AVT :one:
> Page: [Guidelines - Pictograms: Usage](https://www.carbondesignsystem.com/guidelines/pictograms/usage)
> This bug affects 15 other issues

## Detailed Description:
Image does not have an 'alt' attribute short text alternative

<img width="1680" alt="Screen Shot 2020-08-04 at 12 46 48 PM" src="https://user-images.githubusercontent.com/32720851/89321309-cfc8ea00-d650-11ea-8721-96084b5c290a.png">

## What to do:
If the image conveys meaning, use the alt attribute on the <img> element to provide a short description that serves the same purpose as the image.

## Why is this important?
When an image contains important information, providing a text alternative makes the same information accessible through audio or other channels.

**Comment by andreancardona** [comment]
Fix can be seen here: https://github.com/carbon-design-system/carbon-website/pull/2149
